ui/packages/consul-ui/tests/pages/dc/services/show.js

```diff
@@ -26,6 +35,20 @@ export default function(visitable, clickable, attribute, collection, text, inten
     }),
     intentionList: intentions(),
   };
+  page.tabs.topologyTab = {
+    defaultAllowNotice: {
+      see: isPresent('[data-test-notice="default-allow"]'),
+    },
+    filteredByACLs: {
+      see: isPresent('[data-test-notice="filtered-by-ACLs"]'),
```

Comment:
Whoops! I committed this change but it broke tests because the change needed to be made in the hbs file as well. I've committed this changes in a separate commit.